src/cloudwatch/modules/client/baserequestbuilder.py

```diff
@@ -0,0 +1,65 @@
+from ..awsutils import get_aws_timestamp, get_datestamp
+from signer import Signer
+from querystringbuilder import QuerystringBuilder
```

Comment:
Relative imports don't work this way in Python 3+, so you're best off just doing absolute imports:

```python
from cloudwatch.modules.awsutils import get_aws_timestamp, get_datestamp
from .signer import Signer
...
```

I specifically do an absolute import for the awsutils because it's out-of-module. Relative imports in the same module can use `.` pretty painlessly.